openapi/openapi.yml

```diff
@@ -1370,15 +1327,19 @@ components:
       schema:
         type: integer
         default: 10
-    postgres_database_id:
-      description: Postgres database ID
+    postgres_database_identifier:
+      description: Postgres database ID or name
       in: path
-      name: postgres_database_id
+      name: postgres_database_identifier
       required: true
       schema:
-        type: string
-        example: pgn30gjk1d1e2jj34v9x0dq4rp
-        pattern: '^pg[0-9a-hj-km-np-tv-z]{24}$'
+        oneOf:
+          - type: string
+            example: pgn30gjk1d1e2jj34v9x0dq4rp
+            pattern: '^pg[0-9a-hj-km-np-tv-z]{24}$'
+          - type: string
+            example: postgres_database_name
+            pattern: '^(?:(?!pg)).{1,26}$'
```

Comment:
The allowed name pattern is in `Validation::ALLOWED_NAME_PATTERN`.  It does not allow underscores, so the example needs to be fixed.

Reply:
I adjusted it to match that validation here: https://github.com/ubicloud/ubicloud/pull/2907/commits/fb5946d1332379d1542ba6c48ee2418987b6bdb1

Note, I think this technically is not quite right as it ought to specifically omit the case where it begins with pg and is 26 characters, but my regex-foo is at least presently failing me in terms of how to write that.